sdk/identity/Azure.Identity/src/AzureCliCredential.cs

```diff
@@ -130,6 +131,13 @@ private async ValueTask<AccessToken> RequestCliAccessTokenAsync(bool async, stri
                     throw new CredentialUnavailableException(AzNotLogIn);
                 }
 
+                bool isRefreshTokenFailedError = exception.Message.IndexOf(AzureCliFailedError, StringComparison.OrdinalIgnoreCase) != -1 && exception.Message.IndexOf("The provided authorization code or refresh token has expired due to inactivity. Send a new interactive authorization request for this user and resource.", StringComparison.OrdinalIgnoreCase) != -1;
```

Comment:
nit: should this new string be in a constant as well?

Reply:
I didn't create one just because it is only used here. It wouldn't hurt though.